src/Runtime/Runtime.php

```diff
@@ -55,11 +55,11 @@ public function run($argv): int
     protected function doRun($argv, $output): int
     {
         // Do the preflight steps
-        $status = $this->preflight->preflight($argv);
+        $preflightSuccess = $this->preflight->preflight($argv);
```

Comment:
`false` does not indicate failure, so I would rename this something like `$preflightDidRedispatch`

Reply:
Thanks for the feedback, this should be fixed now.